Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -21,13 +21,13 @@
 namespace AzFramework
 {
     template<typename T>
-    void SpawnableEntitiesManager::QueueRequest(EntitySpawnTicket& ticket, SpawnablePriority priority, T&& request)
+    void SpawnableEntitiesManager::QueueRequest(Ticket* ticket, SpawnablePriority priority, T&& request)
```

Comment:
This is using a void* ticket because when the entity gets added to the game context, it only has access to void* and not the EntitySpawnTicket. 
If we use a map of <TicketId, Ticket*> like suggested in the other comment, then this can use EntitySpawnTicket& like the others. Will do that once a decision is reached on the other thread on whether we should use lookup tables or not.

Reply:
The alternative approach I suggested doesn't suffer from this issue and keep the overall API cleaner and consistent.